src/template-tags/tickets.php

```diff
@@ -403,6 +403,48 @@ function tribe_events_ticket_is_on_sale( Tribe__Tickets__Ticket_Object $ticket )
 	}
 }//end if
 
+if ( ! function_exists( 'tribe_tickets_is_current_time_in_date_window' ) ) {
+
+	/**
+	 * Checks if the post has tickets that are available in the current date range set on the ticket.
+	 *
+	 * @since TBD
+	 *
+	 * @param int $post_id Post (or event) to check for ticket availability.
+	 *
+	 * @return bool
+	 */
+	function tribe_tickets_is_current_time_in_date_window( $post_id ) {
+		static $ticket_availability = [];
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 838a6f85b18802dbd16cb727635284b0</sub>

Reply:
Lies and untruths.

Reply:
Oh, that is just wrong - bot needs update?